src/seedu/addressbook/data/person/NameTest.java

```diff
@@ -33,6 +33,8 @@ public void setup() {
     @Test
     public void test() {
```

Comment:
please follow the naming convention for tests; hope that the tutorial discussions were helpful in identifying some common errors like this